docs/setup/upgrade/upgrade-standard.asciidoc

```diff
@@ -1,40 +1,40 @@
 [[upgrade-standard]]
-=== Standard Upgrade
+=== Standard upgrade
```

Comment:
Something to consider... since there's no longer a non-standard upgrade, you could potentially split this content into two pages to match the pages under "Installing Kibana".  For example:

Set Up Kibana
-- Installating Kibana
----Install Kibana with .tar.gz
----Install Kibana with Debian Package
...
-- Upgrading Kibana
---- Upgrade Kibana with .tar.gz
---- Upgrade Kibana with RPM or Debian package